src/app/tests/suites/certification/Test_TC_WNCV_3_1.yaml

```diff
@@ -91,11 +87,11 @@ tests:
 
     ### Optional Depends on the FeatureMap
     - label:
-          "1f: If (PA & TL) TH reads CurrentPositionTiltPercentage optional
-          attribute from DUT"
+          "1f: TH reads CurrentPositionTiltPercentage optional attribute from
+          DUT"
       command: "readAttribute"
       attribute: "CurrentPositionTiltPercentage"
-      PICS: WNCV.S.F01 && WNCV.S.F04 && WNCV.S.A0009
+      PICS: WNCV.S.A0009 && ( WNCV.S.F00 || WNCV.S.F01 )
```

Comment:
wrong

Reply:
Removed changes from PR please verify it.